### DISCUSSION TITLE
Add Checkbox to GiveWP forms

### DISCUSSION THREAD

**Discussion by ClickCoSupport** [impress-org/givewp#7330] [Developer Q&A]
Is there a snippet to add a checkbox to GiveWP forms as is available for textbox and select fields? 

This link has the snippet for textbox and select fields:
 https://givewp.com/documentation/developers/how-to-create-custom-form-fields

**Comment by AndreaSTL** [comment]
Hi, i go on in this thread since i've a similar question.

I need to add a donation checkbox just under the "accept term & condition" field and i need the same graphic of the "terms and condition" checkbox, with a rectangular checkbox with the option to show/hide a text.

The need is to accept marketing GPDR conditions and it's should be optional (not required).

Do you have any suggestion on how to do it?

![image](https://github.com/impress-org/givewp/assets/165407105/cd165386-8393-4fec-a030-9c012d5decca)
